src/main/java/beam/playground/point2point/PointToPointMain.java

```diff
@@ -0,0 +1,37 @@
+package beam.playground.point2point;
+
+import com.conveyal.r5.transit.TransportNetwork;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * Created by ahmar.nadeem on 5/28/2017.
+ */
+public class PointToPointMain {
```

Comment:
Shouldn't the name to be more specific, e.g; NetworkBuilder or something that reflect the purpose.